### PR TITLE
Clarify blog post

### PR DESCRIPTION
**Pull request by mizzle-mo**
Is snabdom still supported?  It's mentioned in the post, but I saw in the chat where it looks like maybe it is no longer supported?

### CODE REVIEW

**Comment by forki** [comment]
❤️

**Comment by MangelMaxime** [comment]
We removed snabbdom support from elmish. 

Explanations are here: https://github.com/fable-compiler/fable-elmish/issues/62

File: docs/source/blog/Introducing-1-0-beta.md [hunk 1]
```diff
@@ -133,6 +133,18 @@ dotnet fable add fable-powerpack@next fable-react@next fable-elmish@next fable-e
...
 > At the moment Fable cannot resolve transitive dependencies so it's necessary to list them all.
 
+#### Important Note
+
+When adding new dependencies and making changes to the project file, it is often necessary to run the following commands again.
+
+```shell 
+ dotnet restore
+ dotnet build
+ dotnet fable npm-run build
```
Comment (alfonsogarciacaro) [inline docs/source/blog/Introducing-1-0-beta.md]: In theory, with latest Ionide & FSharp.Net.Sdk you shouldn't need to run `dotnet build`. I need to check. Also, maybe we can take `dotnet fable npm-run build` out of this list as it's not directly related with intellisense and users may want to use another npm-script (e.g., `start`).

**Comment by alfonsogarciacaro** [comment]
Thanks @mizzle-mo and @MangelMaxime! I'll add a couple of other changes I wanted to make and update the blog post :+1:

File: docs/source/blog/Introducing-1-0-beta.md (hunk 1 above)
Comment (MangelMaxime) [inline docs/source/blog/Introducing-1-0-beta.md]: Personally I never use `dotnet build` myself.

**Comment by mizzle-mo** [comment]
@alfonsogarciacaro Yeah, I was throwing the kitchen sink at it each time, because I could never remember which command fixed it.

**Pull request merged**